pkg/controllers/termination/eviction.go

```diff
@@ -85,17 +90,23 @@ func (e *EvictionQueue) Start(ctx context.Context) {
 	logging.FromContext(ctx).Errorf("EvictionQueue is broken and has shutdown")
 }
 
-// evict returns true if successful eviction call, error is returned if not eviction-related error
+// evict returns true if successful eviction call, and false if not an eviction-related error
 func (e *EvictionQueue) evict(ctx context.Context, nn types.NamespacedName) bool {
 	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).With("pod", nn))
 	err := e.coreV1Client.Pods(nn.Namespace).Evict(ctx, &v1beta1.Eviction{
 		ObjectMeta: metav1.ObjectMeta{Name: nn.Name, Namespace: nn.Namespace},
 	})
+	// status codes for the eviction API are defined here:
+	// https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/#how-api-initiated-eviction-works
 	if errors.IsNotFound(err) { // 404
 		return true
 	}
-	if errors.IsTooManyRequests(err) { // 429
+	if errors.IsTooManyRequests(err) { // 429 - PDB violation
 		logging.FromContext(ctx).Debug(err)
```

Comment:
Do we still want to log since the event emitter logs?

Reply:
nope, didn't notice until I was testing on EKS :D